google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -20,7 +20,8 @@ cloud.project.label=Project\:
 
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
-appengine.cloudsdk.location.warning.message=Please select a Cloud SDK directory.
+appengine.cloudsdk.location.missing.message=Please select a Cloud SDK directory.
```

Comment:
depending on how these two messages appear in the UI you might want a colon instead of a period